Quantum-Control-Applications/AMO/Use Case 1 - Saffman Lab - Atom Sorting/Array_sorting.py

```diff
@@ -2,14 +2,14 @@
 from qm.QuantumMachinesManager import QuantumMachinesManager
 from qm import SimulationConfig
 from qm import LoopbackInterface
-from Config_array_sorting import *
+from config_array_sorting import *

```

Comment:
you also need to rename the file(s)

Reply:
I did but for some reason github did not update them...

Reply:
Done